ami/ansible/roles/awslogs/files/awslogs.conf

```diff
@@ -0,0 +1,64 @@
+#
+# ------------------------------------------
+# CLOUDWATCH LOGS AGENT CONFIGURATION FILE
+# ------------------------------------------
+#
+# --- DESCRIPTION ---
+# This file is used by the CloudWatch Logs Agent to specify what log data to send to the service and how.
+# You can modify this file at any time to add, remove or change configuration.
+#
+# NOTE: A running agent must be stopped and restarted for configuration changes to take effect.
+#
+# --- CLOUDWATCH LOGS DOCUMENTATION ---
+# https://aws.amazon.com/documentation/cloudwatch/
+#
+# --- CLOUDWATCH LOGS CONSOLE ---
+# https://console.aws.amazon.com/cloudwatch/home?region=us-east-1#logs:
+#
+# --- AGENT COMMANDS ---
+# To check or change the running status of the CloudWatch Logs Agent, use the following:
+#
+# To check running status: /etc/init.d/awslogs status
+# To stop the agent: /etc/init.d/awslogs stop
+# To start the agent: /etc/init.d/awslogs start
+#
+# --- AGENT LOG OUTPUT ---
+# You can find logs for the agent in /var/log/awslogs.log
+# You can find logs for the agent script in /var/log/awslogs-agent-setup.log
+#
+# ------------------------------------------
+# CONFIGURATION DETAILS
+# ------------------------------------------
+# Refer to http://docs.aws.amazon.com/AmazonCloudWatch/latest/DeveloperGuide/AgentReference.html for details.
+
+[general]
+state_file = /var/awslogs/state/agent-state
+logging_config_file = /etc/awslogs/awslogs.conf
```

Comment:
I think this needs to be
logging_config_file = /var/etc/awslogs/awslogs.conf
as you installed via script, and not via rpm; see https://docs.aws.amazon.com/AmazonCloudWatch/latest/logs/AgentReference.html

Reply:
Since the ZK instance is assuming an IAM role, this won't really matter. I'll just remove that line for now since it's optional anyway.